theme.json

```diff
@@ -2,6 +2,49 @@
 	"settings": {
 		"appearanceTools": true,
 		"color": {
+			"defaultDuotone": false,
+			"duotone": [
+				{
+					"colors": [
+						"#111111",
+						"#ffffff"
+					],
+					"slug": "black-and-white",
+					"name": "Duotone 1"
```

Comment:
I think this would make more sense to follow the slug, since it's the one the user will see

Reply:
Good catch! Done.